src/pages/docs/main-concepts/2. assignments.md

```diff
@@ -0,0 +1,30 @@
+---
+path: "/assignments"
+date: 2019-06-24T22:09:49.849Z
+title: "Assignments"
+---
+
+Now that we have an understanding of the various ordercloud parties (companies and groups) we can begin to talk about assignments. When you save an assignment you are creating a relationship between a party and an object (ordercloud "thing" such as product, category, etc). For example you might assign a usergroup to a category, thereby granting a user in that group visibility to that category.
+
+There are basic principles around how assignments work that are critical to understanding our data model and more importantly how the data model can be applied to solve the most complex ordering scenarios efficiently
+- Assignments are used to define a relationship
+- Assignments are inclusive
+- Assignments can be made at different levels
+- Assignments cascade down higher levels to the individual user
+- Assignments are many to many
+
+## Assignments are inclusive
```

Comment:
Let's standardize title casing here - "Assignments are Inclusive"

Reply:
How would this one look with the standardized casing?
Assignments can be made at different levels

Assignments can be Made at Different Levels ?

Reply:
Yeah I believe so